pkg/kubectl/explain/field_lookup_test.go

```diff
@@ -69,9 +69,9 @@ func TestFindField(t *testing.T) {
 			gotPath = path.GetPath().String()
 		}
 
-		if gotErr != test.err && gotPath != test.expectedPath {
-			t.Errorf("LookupSchemaForField(schema, %v) = (%v, %v), expected (%s, %v)",
-				test.path, gotErr, gotPath, test.expectedPath, test.err)
+		if gotErr != test.err || gotPath != test.expectedPath {
+			t.Errorf("LookupSchemaForField(schema, %v) = (path: %v, err: %v), expected (path: %s, err: %v)",
```

Comment:
maybe use `%q`? what do you think?

Reply:
I see one other thing that I have `path: %v` and `path: %s`, though both seems producing consistent formatting here, it's good to have both as v%?. About %q, let's see what we come with our offline discussion :-). Thanks!!